app/classes/Comment/index.js

```diff
@@ -0,0 +1,7 @@
+const Comment = require('./Comment')
+
+// singleton
+const instance = new Comment()
+Object.freeze(instance)
```

Comment:
@ksraj123 No need to freeze as you are exporting a single instance of the class

Reply:
I am freezing this because I think we do want the public methods to be able to be changed or new stuff to be added to the object, we want it to stay like as its defined in the class.

Reply:
Oh okay cool!
Got it. You mean that putting the restriction on overwriting right?

Reply:
Yes, and upon adding more stuff to the object as well.